packages/data-stores/src/wpcom-request-controls/index.ts

```diff
@@ -53,4 +55,6 @@ export const controls = {
 		triggerReloadProxy();
 	},
 	REQUEST_ALL_BLOGS_ACCESS: () => triggerRequestAllBlogsAccess(),
+	WAIT: ( { ms }: ReturnType< typeof wait > ) =>
+		new Promise( resolve => setTimeout( resolve, ms ) ),
```

Comment:
Just wanted to make a note that this feels like a really elegant way to introduce a wait! 👍